tensorflow/core/api_def/base_api/api_def_AssertNextDataset.pbtxt

```diff
@@ -1,4 +1,32 @@
 op {
   graph_op_name: "AssertNextDataset"
   visibility: HIDDEN
+  in_arg {
+    name: "input_dataset"
+    description: <<END
+A variant tensor representing the input dataset.
+`AssertNextDataset` passes through the outputs of its input dataset.
+END
+  }
+  in_arg {
+  name: "transformations"
+  description: <<END
+A `tf.string` vector `tf.Tensor` identifying the transformations that are
+expected to happen next.
+END
+  }
+  summary: "A transformation that asserts which transformations happen next."
+  description: <<END
+Core C++ implementation of the transformation `assert_next()`, which is
```

Comment:
please remove this paragraph

Reply:
Thanks for the follow-on review. Commit d4ab78b removes the first paragraph of the description.